float64 truWndDir
float64 truWndSpd
float64 cog
float64 sog
float64 VMG
float64 VMGup
float64 XTE

